www/js/input-page.js

```diff
@@ -648,7 +671,24 @@ var toolTip = function(toolTipLocation){
 //Calls submitArthropodsToServer if survey upload is successful
 var submitSurveyToServer = function(){
 //	navigator.notification.alert("Submitting survey");
-	$.ajax({
+	var online = navigator.onLine;
+	if(online == false){
+		//alert("Insdie of submit Survey function");
+		db.transaction(function(tx){
+                        tx.executeSql("INSERT INTO SURVEY VALUES (?,?,?,?,?,?,?,?,?,?,?,?,?,?,?)", ['survey','123','MattWu',"9406","2","C","08:05","40","49","note","bird","1","Normal_Type","5","Mobile"]);
+						//alert("survey database inside");
```

Comment:
you should pull the value from each user input and push them into the database, not faking an entry.